### PR TITLE
PANDARIA: Add chart fluent-bit

### PR DESCRIPTION
**Pull request by GGGitBoy**
支持使用fluent-bit进行日志采集

**Relate issue：**
https://github.com/cnrancher/pandaria/issues/1163

### CODE REVIEW

File: charts/fluent-bit/README.md [hunk 1]
```diff
@@ -0,0 +1 @@
+# fluent-bit
```
Comment (guangbochen) [inline charts/fluent-bit/README.md:1]: add the README file with some proper description, u may refer to the official fluent-bit chart.

**Review by guangbochen — COMMENTED** [review]
@GGGitBoy is there any difference between https://github.com/fluent/helm-charts/tree/master/charts/fluent-bit and the one we have added?

**Comment by GGGitBoy** [comment]
@guangbochen some differences like default log message, YAML format, etc.

**Review by guangbochen — APPROVED** [review]
LGTM

**Pull request merged**